src/LibChorus/FileTypeHandlers/text/TextFileTypeHandler.cs

```diff
@@ -64,26 +66,36 @@ public static string GetRawMerge(string oursPath, string commonPath, string thei
 		{
 			//NB: surrounding with quotes didn't cut it to get past paths with spaces
 
-			return RunProcess("diff3/bin/diff3.exe", "-m " + LongToShortConverter.GetShortPath(oursPath) + " " +
-				LongToShortConverter.GetShortPath(commonPath) + " " +
-				LongToShortConverter.GetShortPath(theirPath));
+			return RunProcess(_diff3Exe, new string[] {"-m", LongToShortConverter.GetShortPath(oursPath),
+				LongToShortConverter.GetShortPath(commonPath),
+				LongToShortConverter.GetShortPath(theirPath) });
 		}
 
 		protected static string SurroundWithQuotes(string path)
 		{
 			return "\"" + path + "\"";
 		}
 
-		public static string RunProcess(string filePath, string arguments)
+		public static string RunProcess(string filePath, string[] arguments)
 		{
 			Process p = new Process();
+			ProcessStartInfo startInfo = new ProcessStartInfo();
```

Comment:
this startInfo isn't being used as far as I can tell

Reply:
Yep, that was leftover from an earlier attempt to solve the bug with quoting filenames with spaces. I'll remote it.

Reply:
Done in commit 3b4f7d1b.